app/mailers/request_mailer.rb

```diff
@@ -0,0 +1,24 @@
+class RequestMailer < ApplicationMailer
+  def request_cancel_partner_notification(request_id:)
+    @request ||= Request.find(request_id)
+    @organization ||= @request.organization
+    @partner = @request.partner
+
+    # Generate the request_items in a formatted way so
+    # I can render Item names
+    item_ids = @request.request_items.map { |r| r['item_id'] }
+    items = Item.where(id: item_ids)
+    @formatted_requested_items = @request.request_items.map do |rt|
+      {
+        name: items.find { |i| i.id == rt['item_id'] }&.name,
+        quantity: rt['quantity']
+      }
+    end
+    @formatted_requested_items.sort_by! { |rt| rt[:name] }
```

Comment:
I'm being super nitpicky and OCD, but using `rt` as the block variable in both places makes me think they're the same shape, but in the first block they are `@request.request_items` (i.e. hashes with `'item_id'` and `'quantity'` keys)  and the second they are the object you just made (hashes with `:name` and `:quantity` keys). Should it be `ri` for the first block (for request_items) and `fri` for the second block (for formatted_request_items)?

Reply:
All good :). I can change the `rt` in the second block to `frt`